README.md

```diff
@@ -235,6 +235,10 @@ fastify.post('/upload/files', async function (req, reply) {
   const body = Object.fromEntries(
     Object.keys(req.body).map((key) => [key, req.body[key].value])
   ) // Request body in key-value pairs, like req.body in Express (Node 12+)
+
+  // On Node 18+
+  const formData = await req.formdata()
```

Comment:
Awesome addition, literally what I wanted to do in the rewrite

Why `req.formdata` instead of `req.formData`?